habitat/plan.sh

```diff
@@ -18,7 +18,8 @@ pkg_build_deps=(
   core/git
 )
 pkg_bin_dirs=(bin)
-pkg_svc_run="bin/dcob"
+pkg_svc_run="bin/dcob -o 0.0.0.0"
```

Comment:
It appears that this command should be `dcob` not `bin/dcob`. The former picks up the binstub for the command that lives on the Habitat service's PATH. The latter looks for a directory `bin/` that does not exist in the default service working directory.